src/rbd_fuse/rbd-fuse.cc

```diff
@@ -877,13 +865,59 @@ connect_to_cluster(rados_t *pcluster)
 	return r;
 }
 
+int init_cluster(int argc, char **argv, char **remargv)
+{
+	int r, remargc;
+
+	r = rados_create(&cluster, NULL);
+	if (r < 0) {
+		simple_err("Could not create cluster handle", r);
+		exit(1);
+	}
+
+	r = rados_conf_parse_argv_remainder(cluster, argc, (const char **)
+						argv, (const char **) remargv);
+	if (r < 0) {
+		simple_err("Error parsing the arguments", r);
+		exit(1);
+	}
+
+	for (remargc = 0; remargv[remargc] != NULL and remargc < argc;
+		++(remargc))
+		;
+
+	return remargc;
+}
+
+void post_init_cluster()
+{
+	int r;
+
+	r = rados_conf_read_file(cluster, rbd_options.ceph_config);
+	if (r < 0) {
+		simple_err("Error reading Ceph config file", r);
+		exit(1);
+	}
+
```

Comment:
I wanted additional `rados_conf_parse_argv` after `rados_conf_read_file` to make command line options override config options (which is expected behavior).

Assuming you have "admin socket" and "log file" options specified in ceph.conf, try this:
```
./bin/rbd-fuse --log-file=/tmp/rbd-fuse.log --admin-socket=/tmp/rbd-fuse.asok  /mnt
```
Your current version will still use the config file options.